packages/KEmptyState/KEmptyState.vue

```diff
@@ -65,22 +68,22 @@ export default {
 </script>
 
 <style scoped>
-  .empty-state-wrapper {
-  padding: 4rem 0;
+.empty-state-wrapper {
+  padding: 42px 0;
   text-align: center;
-  color: rgba(0,0,0,.70);
-  border: 1px solid rgba(151,151,151,.1);
   border-radius: 3px;
-  background-color: rgba(0,0,0,.03);
+  background-color: var(--KEmptyBackground, var(--white));
 }
 .empty-state-wrapper h5 {
-  margin: 0 0 .75rem;
+  color: var(--KEmptyTitleColor, var(--black-500));
+  margin: 0 0 14px;
```

Comment:
Any specific reason for why we are switching from rem to px?

Reply:
This is a good question. For awhile `rem` was necessary due to `px` not scaling well, but now most browsers use zoom to increase scale so rem is actually no longer necessary. The first answer here has a pretty good explanation: https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css